README.md

```diff
@@ -7,4 +7,26 @@ You can see it running at [http://spreedly-endangered-pets.herokuapp.com](http:/
 
 It uses the [httparty](https://github.com/jnunemaker/httparty) gem.
 
+## Running
 
+A free Spreedly [developer test account](https://id.spreedly.com/signup) is
+required for this sample application to interact with the Spreedly API.
+
+Requirements:
+
+- [Postgres](http://postgresapp.com/) running locally
+
+To run:
+
+```
+$ git clone https://github.com/spreedly/sample-endangered-pets.git
+$ cd sample-endangered-pets
+$ bundle install
```

Comment:
Interestingly enough, I get the same error when I `bundle` in core. Seems like there is something jacked with my Postgres.app setup. I was able to get around this error by installing postgres via homebrew.